src/Symfony/Component/PropertyInfo/Util/PhpDocTypeHelper.php

```diff
@@ -164,6 +164,6 @@ private function getPhpTypeAndClass(string $docType): array
             return ['object', $docType];
         }
 
-        return ['object', substr($docType, 1)];
+        return ['object', ltrim($docType, '\\')]; // strip the namespace's `\`-prefix
```

Comment:
please remove the comment at the end of the line

Reply:
Done. For me it's obvious, but [someone added the comment in later branches](https://github.com/symfony/symfony/blob/5b46cb9f0fb5f654f48eb3830e3bda2509bf0eb9/src/Symfony/Component/PropertyInfo/Util/PhpDocTypeHelper.php#L190) so I just repeated it.